drivers/gke/gke_driver.go

```diff
@@ -200,6 +206,120 @@ func (d *Driver) GetDriverCreateOptions(ctx context.Context) (*types.DriverFlags
 		Type:  types.StringType,
 		Usage: "When to performance updates on the nodes, in 24-hour time (e.g. \"19:00\")",
 	}
+	driverFlag.Options["resource-labels"] = &types.Flag{
+		Type:  types.StringSliceType,
+		Usage: "The map of Kubernetes labels (key/value pairs) to be applied to each cluster",
+	}
+	driverFlag.Options["enable-nodepool-autoscaling"] = &types.Flag{
+		Type:  types.BoolType,
+		Usage: "Enable nodepool autoscaling",
+	}
+	driverFlag.Options["min-node-count"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "Minimmum number of nodes in the NodePool. Must be >= 1 and <= maxNodeCount",
+	}
+	driverFlag.Options["max-node-count"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "Maximum number of nodes in the NodePool. Must be >= minNodeCount. There has to enough quota to scale up the cluster",
+	}
+	driverFlag.Options["enable-auto-upgrade"] = &types.Flag{
+		Type:  types.BoolType,
+		Usage: "Specifies whether node auto-upgrade is enabled for the node pool",
+	}
+	driverFlag.Options["enable-auto-repair"] = &types.Flag{
+		Type:  types.BoolType,
+		Usage: "Specifies whether the node auto-repair is enabled for the node pool",
+	}
+	driverFlag.Options["local-ssd-count"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "he number of local SSD disks to be attached to the node",
+	}
+	driverFlag.Options["preemptible"] = &types.Flag{
+		Type:  types.BoolType,
+		Usage: "Whether the nodes are created as preemptible VM instances",
+	}
+	driverFlag.Options["disk-type"] = &types.Flag{
+		Type:  types.StringType,
+		Usage: "Type of the disk attached to each node",
+	}
+	driverFlag.Options["taints"] = &types.Flag{
```

Comment:
I can't find where this value is used, can you point me to the line number?

Reply:
Actually, this value isn't used because there is no this field `taints` in SDK of `google-api-go-client`, So I'll leave it as a reserved field for now. But just now I check the package of `google-api-go-client`, they have added this field in the master branch, so I will rebase this package in vendor to make it useful.

Reply:
Ok, I'll review the change as soon as you push it.

Reply:
I've rebased `google-api-go-client` SDK, so please review it again.